gnomad/utils/vcf.py

```diff
@@ -144,6 +145,14 @@
             " heterozygous genotypes expected under Hardy-Weinberg equilibrium)"
         ),
     },
+    "inbreeding_coeff": {
+        "Number": "A",
+        "Description": (
+            "Inbreeding coefficient, the excess heterozygosity at a variant site,"
+            " computed as 1 - (the number of heterozygous genotypes)/(the number of"
+            " heterozygous genotypes expected under Hardy-Weinberg equilibrium)"
+        ),
+    },
```

Comment:
after L166, you should add "SB", because it's in SITE_FIELDS as in L79. Something like this? 
```
"SB": {
        "Description": (
            "Per-sample component statistics which comprise the Fisher's exact test to detect strand bias. Values are: depth of reference allele on forward strand, depth of reference allele on reverse strand, depth of alternate allele on forward strand, depth of alternate allele on reverse strand."
        )
    },
```

Reply:
I'm not sure what number=?, but the type is not Integar for sure, has to be float, decided directly by hail export, right?

Reply:
Yes -- I noted this in the notebook and slack. I'm trying to find the correct definition, it is not per-sample.

Reply:
Sorry, I misread the numbers yesterday, they are integers, and 4 values.
I found this GATK definition might helps: 
https://gatk.broadinstitute.org/hc/en-us/articles/4409917393307-StrandBiasBySample

Reply:
Ah, Okay, I found it's defined in FORMAT_dict, L516-L524, but we don't have per-sample entries, I guess it's a sum of REF/ALT read counts across all the samples to make this SB in the INFO_dict (SITE_FIELDS)?

Reply:
We're dropping SB from the VCF so not going to add the definition here.